tree/promises.cf

```diff
@@ -76,6 +76,9 @@ body agent control {
     # Repository where to put the copy of modified files
     default_repository => "${sys.workdir}/modified-files";
 
+    # Standard abort class usable in ncf
+    # The agent will stop execution with a fatal error as soon as it is defined
+    abortclasses => { "abort_agent_run" };
```

Comment:
Because it is named `should_not_continue` which is a pretty bad name for something that stops the agent immediately with a fatal error.